sdgym/benchmark.py

```diff
@@ -253,157 +267,139 @@ def _run_on_dask(jobs, verbose):
-        workers (int or str):
-            If ``workers`` is given as an integer value other than 0 or 1, a multiprocessing
-            Pool is used to distribute the computation across the indicated number of workers.
-            If ``workers`` is -1, the number of workers will be automatically determined by
-            the number of GPUs (if available) or the number of CPU cores. If the string ``dask``
-            is given, the computation is distributed using ``dask``. In this case, setting up the
-            ``dask`` cluster and client is expected to be handled outside of this function.
-        cache_dir (str):
-            If a ``cache_dir`` is given, intermediate results are stored in the indicated directory
-            as CSV files as they get computted. This allows inspecting results while the benchmark
-            is still running and also recovering results in case the process does not finish
-            properly. Defaults to ``None``.
+        synthesizers (list[class]):
+            The synthesizer(s) to evaluate. Defaults to ``[GaussianCopulaSynthesizer, FASTMLPreset,
+            CTGANSynthesizer]``. The available options are:
+
+                - ``GaussianCopulaSynthesizer``
+                - ``CTGANSynthesizer``
+                - ``CopulaGANSynthesizer``
+                - ``TVAESynthesizer``
+                - ``FASTMLPreset``
+                - any custom created synthesizer or variant
+
+        sdv_datasets (list[str] or ``None``):
+            Names of the SDV demo datasets to use for the benchmark. Defaults to
+            ``[adult, alarm, census, child, expedia_hotel_logs, insurance, intrusion, news,
+            covtype]``. Use ``None`` to disable using any sdv datasets.
+        additional_datasets_folder (str or ``None``):
+            The path to a folder (local or an S3 bucket). Datasets found in this folder are
+            run in addition to the SDV datasets. If ``None``, no additional datasets are used.
+        limit_dataset_size (bool):
+            Use this flag to limit the size of the datasets for faster evaluation. If ``True``,
+            limit the size of every table to 1,000 rows (randomly sampled) and the first 10
+            columns.
+        evaluate_quality (bool):
+            Whether or not to evaluate an overall quality score.
+        sdmetrics (list[str]):
+            A list of the different SDMetrics to use. If you'd like to input specific parameters
+            into the metric, provide a tuple with the metric name followed by a dictionary of
+            the parameters.
+        timeout (bool or ``None``):
+            The maximum number of seconds to wait for synthetic data creation. If ``None``, no
+            timeout is enforced.
+        output_filepath (str or ``None``):
+            A file path for where to write the output as a csv file. If ``None``, no output
+            is written.
+        detailed_results_folder (str or ``None``):
```

Comment:
I think this should be called `intermediary_results_folder`. The issue also seems to suggest that

Reply:
Ah the issue is out of date. I'll correct it there.

Reply:
Is there a reason we changed it? Intermediary seems to make more sense to me

Reply:
I think `intermediate` seemed like it would require the user to understand more about the implementation (these results are collected throughout the run), and `detailed_results_folder` is just a description of what the folder contains.